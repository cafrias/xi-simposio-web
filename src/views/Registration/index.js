import React from 'react'

import './Registration.sass'

// COMPONENTS __________________________________________________________________
import Section from '../../components/Section'

function Registration() {
  return (
    <Section id="about" name="InscripciÃ³n"
      className="background_orange registration-container">
      <p>Merca</p>
    </Section>
  )
}

export default Registration
