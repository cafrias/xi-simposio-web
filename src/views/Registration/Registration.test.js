import React from 'react'
import {shallow} from 'enzyme'

import Registration from './index'

describe('Registration view', () => {
  var wrapper

  beforeEach(() => wrapper = shallow(<Registration />))

  it('should render without errors', () => {
    expect(wrapper).toHaveLength(1)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
