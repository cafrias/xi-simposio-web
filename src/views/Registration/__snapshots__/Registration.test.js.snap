exports[`Registration view should match snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Section
    className="background_white registration-container"
    id="about"
    name="Inscripción">
    <article
        className="deadlines">
        <h2>
            Presentación de trabajos
        </h2>
        <div
            className="deadline">
            <h3
                className="deadline__title">
                Fecha límite presentación de Resúmenes
            </h3>
            <p
                className="table__date deadline__date">
                <span
                    className="table__weekday">
                    Viernes
                </span>
                21/04/2017
            </p>
        </div>
        <div
            className="deadline">
            <h3
                className="deadline__title">
                Fecha límite presentación de Trabajos
            </h3>
            <p
                className="table__date deadline__date">
                <span
                    className="table__weekday">
                    Viernes
                </span>
                05/05/2017
            </p>
        </div>
    </article>
    <article
        className="costs">
        <h2>
            Costo de las matrículas
        </h2>
        <div
            className="cost">
            <h3
                className="cost__title">
                Asistentes en general
            </h3>
            <p
                className="cost__total">
                $2.200*
            </p>
        </div>
        <div
            className="cost">
            <h3
                className="cost__title">
                Matriculados Consejo Profesional de Ciencias Económicas TDF
            </h3>
            <p
                className="cost__total">
                $1.600*
            </p>
        </div>
        <div
            className="cost">
            <h3
                className="cost__title">
                Docentes UNTDF
            </h3>
            <p
                className="cost__total">
                $1.300*
            </p>
        </div>
        <div
            className="cost">
            <h3
                className="cost__title">
                Estudiantes UNTDF
            </h3>
            <p
                className="cost__total">
                Sin cargo
            </p>
        </div>
        <div
            className="cost">
            <h3
                className="cost__title">
                Estudiantes otras instituciones
            </h3>
            <p
                className="cost__total">
                $300**
            </p>
        </div>
        <p
            className="text-container disclaimer">
            <small>
                * Incluye material del Simposio, Cóctail de bienvenida y Cena de camaradería
                <br />
                ** Incluye el material del Simposio
            </small>
        </p>
    </article>
    <article
        className="how-to">
        <h2>
            ¿Cómo inscribirse?
        </h2>
        <p
            className="text-container">
            Llenar 
            <a
                href="www.google.com">
                este
            </a>
             formulario y enviarlo a: 
             
            <a
                href="mailto:ponencias.simposio@gmail.com">
                ponencias.simposio@gmail.com
            </a>
        </p>
    </article>
</Section>,
  "nodes": Array [
    <Section
      className="background_white registration-container"
      id="about"
      name="Inscripción">
      <article
            className="deadlines">
            <h2>
                  Presentación de trabajos
            </h2>
            <div
                  className="deadline">
                  <h3
                        className="deadline__title">
                        Fecha límite presentación de Resúmenes
                  </h3>
                  <p
                        className="table__date deadline__date">
                        <span
                              className="table__weekday">
                              Viernes
                        </span>
                        21/04/2017
                  </p>
            </div>
            <div
                  className="deadline">
                  <h3
                        className="deadline__title">
                        Fecha límite presentación de Trabajos
                  </h3>
                  <p
                        className="table__date deadline__date">
                        <span
                              className="table__weekday">
                              Viernes
                        </span>
                        05/05/2017
                  </p>
            </div>
      </article>
      <article
            className="costs">
            <h2>
                  Costo de las matrículas
            </h2>
            <div
                  className="cost">
                  <h3
                        className="cost__title">
                        Asistentes en general
                  </h3>
                  <p
                        className="cost__total">
                        $2.200*
                  </p>
            </div>
            <div
                  className="cost">
                  <h3
                        className="cost__title">
                        Matriculados Consejo Profesional de Ciencias Económicas TDF
                  </h3>
                  <p
                        className="cost__total">
                        $1.600*
                  </p>
            </div>
            <div
                  className="cost">
                  <h3
                        className="cost__title">
                        Docentes UNTDF
                  </h3>
                  <p
                        className="cost__total">
                        $1.300*
                  </p>
            </div>
            <div
                  className="cost">
                  <h3
                        className="cost__title">
                        Estudiantes UNTDF
                  </h3>
                  <p
                        className="cost__total">
                        Sin cargo
                  </p>
            </div>
            <div
                  className="cost">
                  <h3
                        className="cost__title">
                        Estudiantes otras instituciones
                  </h3>
                  <p
                        className="cost__total">
                        $300**
                  </p>
            </div>
            <p
                  className="text-container disclaimer">
                  <small>
                        * Incluye material del Simposio, Cóctail de bienvenida y Cena de camaradería
                        <br />
                        ** Incluye el material del Simposio
                  </small>
            </p>
      </article>
      <article
            className="how-to">
            <h2>
                  ¿Cómo inscribirse?
            </h2>
            <p
                  className="text-container">
                  Llenar 
                  <a
                        href="www.google.com">
                        este
                  </a>
                   formulario y enviarlo a: 
                   
                  <a
                        href="mailto:ponencias.simposio@gmail.com">
                        ponencias.simposio@gmail.com
                  </a>
            </p>
      </article>
</Section>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Registration />,
      "_debugID": 3,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 2,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Section
          className="background_white registration-container"
          id="about"
          name="Inscripción">
          <article
                    className="deadlines">
                    <h2>
                              Presentación de trabajos
                    </h2>
                    <div
                              className="deadline">
                              <h3
                                        className="deadline__title">
                                        Fecha límite presentación de Resúmenes
                              </h3>
                              <p
                                        className="table__date deadline__date">
                                        <span
                                                  className="table__weekday">
                                                  Viernes
                                        </span>
                                        21/04/2017
                              </p>
                    </div>
                    <div
                              className="deadline">
                              <h3
                                        className="deadline__title">
                                        Fecha límite presentación de Trabajos
                              </h3>
                              <p
                                        className="table__date deadline__date">
                                        <span
                                                  className="table__weekday">
                                                  Viernes
                                        </span>
                                        05/05/2017
                              </p>
                    </div>
          </article>
          <article
                    className="costs">
                    <h2>
                              Costo de las matrículas
                    </h2>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Asistentes en general
                              </h3>
                              <p
                                        className="cost__total">
                                        $2.200*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Matriculados Consejo Profesional de Ciencias Económicas TDF
                              </h3>
                              <p
                                        className="cost__total">
                                        $1.600*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Docentes UNTDF
                              </h3>
                              <p
                                        className="cost__total">
                                        $1.300*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Estudiantes UNTDF
                              </h3>
                              <p
                                        className="cost__total">
                                        Sin cargo
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Estudiantes otras instituciones
                              </h3>
                              <p
                                        className="cost__total">
                                        $300**
                              </p>
                    </div>
                    <p
                              className="text-container disclaimer">
                              <small>
                                        * Incluye material del Simposio, Cóctail de bienvenida y Cena de camaradería
                                        <br />
                                        ** Incluye el material del Simposio
                              </small>
                    </p>
          </article>
          <article
                    className="how-to">
                    <h2>
                              ¿Cómo inscribirse?
                    </h2>
                    <p
                              className="text-container">
                              Llenar 
                              <a
                                        href="www.google.com">
                                        este
                              </a>
                               formulario y enviarlo a: 
                               
                              <a
                                        href="mailto:ponencias.simposio@gmail.com">
                                        ponencias.simposio@gmail.com
                              </a>
                    </p>
          </article>
</Section>,
        "_debugID": 4,
        "_renderedOutput": <Section
          className="background_white registration-container"
          id="about"
          name="Inscripción">
          <article
                    className="deadlines">
                    <h2>
                              Presentación de trabajos
                    </h2>
                    <div
                              className="deadline">
                              <h3
                                        className="deadline__title">
                                        Fecha límite presentación de Resúmenes
                              </h3>
                              <p
                                        className="table__date deadline__date">
                                        <span
                                                  className="table__weekday">
                                                  Viernes
                                        </span>
                                        21/04/2017
                              </p>
                    </div>
                    <div
                              className="deadline">
                              <h3
                                        className="deadline__title">
                                        Fecha límite presentación de Trabajos
                              </h3>
                              <p
                                        className="table__date deadline__date">
                                        <span
                                                  className="table__weekday">
                                                  Viernes
                                        </span>
                                        05/05/2017
                              </p>
                    </div>
          </article>
          <article
                    className="costs">
                    <h2>
                              Costo de las matrículas
                    </h2>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Asistentes en general
                              </h3>
                              <p
                                        className="cost__total">
                                        $2.200*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Matriculados Consejo Profesional de Ciencias Económicas TDF
                              </h3>
                              <p
                                        className="cost__total">
                                        $1.600*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Docentes UNTDF
                              </h3>
                              <p
                                        className="cost__total">
                                        $1.300*
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Estudiantes UNTDF
                              </h3>
                              <p
                                        className="cost__total">
                                        Sin cargo
                              </p>
                    </div>
                    <div
                              className="cost">
                              <h3
                                        className="cost__title">
                                        Estudiantes otras instituciones
                              </h3>
                              <p
                                        className="cost__total">
                                        $300**
                              </p>
                    </div>
                    <p
                              className="text-container disclaimer">
                              <small>
                                        * Incluye material del Simposio, Cóctail de bienvenida y Cena de camaradería
                                        <br />
                                        ** Incluye el material del Simposio
                              </small>
                    </p>
          </article>
          <article
                    className="how-to">
                    <h2>
                              ¿Cómo inscribirse?
                    </h2>
                    <p
                              className="text-container">
                              Llenar 
                              <a
                                        href="www.google.com">
                                        este
                              </a>
                               formulario y enviarlo a: 
                               
                              <a
                                        href="mailto:ponencias.simposio@gmail.com">
                                        ponencias.simposio@gmail.com
                              </a>
                    </p>
          </article>
</Section>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Registration />,
}
`;
